pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -619,7 +630,67 @@ func (a *Actuator) UpdateLoadBalancers(client awsclient.Client, providerConfig *
 	}
 
 	if len(errs) > 0 {
-		return fmt.Errorf("failed to register instances with elbs: %v", errs)
+		return fmt.Errorf("failed to register instances with classic ELBs: %v", errs)
```

Comment:
Observation: `...%v", errs` will format as `[foo bar baz]` - presumably that's OK?